packages/components-js/scripts/generateVRTPages.ts

```diff
@@ -177,9 +177,12 @@ const generateVRTPagesForJsFramework = (htmlFileContentMap: Record<string, strin
           ? convertToRemixVRTPage(...baseParams, reactCharacteristics)
           : { fileName: '', fileContent: '' };
 
-      writeFile(path.resolve(pagesDirectories[framework], convertedFileName), convertedFileContent);
-
-      return './' + path.parse(convertedFileName).name;
+      writeFile(
+        path.resolve(path.resolve(rootDirectory, pagesDirectories[framework]), convertedFileName),
```

Comment:
Why resolve `rootDirectory` again? I think this is already resolved as part of `pagesDirectories`.